Norm_end_to_end.Rmd

```diff
@@ -46,22 +45,21 @@ maximum) and all values between them have an equal probability of being
 selected.
 
 ```{r}
-#' Load preprocessing results 
+#' Load preprocessing results
 load("SumExp.RData")
 load("data.RData")
 
 #' Define colors for the groups.
-col_source <- brewer.pal(8, name = "Dark2")[c(5, 4, 3)]
-names(col_source) <- c("QC",           
-                       "CVD",    
-                       "CTR")    
-
-
-#' Impute missing values.
-
-na.unidis <- function(z){ 
+col_phenotype <- brewer.pal(8, name = "Dark2")[c(8, 4, 3)]
```

Comment:
to easily remember the variable name, I usually use `col_` with then the name of the column (covariate) of the phenodata xls sheet (or `colData` or `sampleData`). Also, I usually create a variable `col_sample` that has the correct color for each sample.